### DISCUSSION TITLE
Hardware question

### DISCUSSION THREAD

**Discussion by inferno1231** [awawa-dev/HyperHDR#297] [General]
Sorry everyone as it is not directly related to HyperHDR, but maybe somebody could point me in the right direction.

So, right now my setup looks like this:

Samsung Q70T TV, PS5/Apple TV and Denon X3700H AVR. All my streaming boxes, game consoles etc. are connected to AVR. Right now I am using first AVR OUT port to feed my TV and second OUT port to feed a 4k 60hz splitter (FeinTech VSP01222 to be specific).
-->USB capture card.

Unfortunately Denon can't send different outputs on 2 out ports. So as my splitter is 4k 60 hz max, this is what AVR sends to TV too. 

So when I unplug second HDMI out, which serves LEDs, I can turn on VRR and all this new gaming stuff on PS5, but when I plug it back in I am limited to 4k 60 hz no VRR etc. 

I need some way to trick my AVR into thinking that it outputs 4k 120 HZ on both OUT ports, as far as I know there are currently no splitters nor EDID emulators that support HDMI 2.1 apart from HdFury VrRoom, but HdFury support told me that their product won't work in my setup either.

So right now I can either use my LEDs without VRR, or disable LEDs and turn on all that gaming stuff. There is also a way when I connect PS5 directly to TV and send sound to AVR using EARC, but in that case I also can't capture feed for HyperHDR, so I am losing LEDs functionality for PS5 but it will still work for Apple TV etc.

Any ideas? Thank you.

**Comment by awawa-dev** [comment]
Hi
So you can't force HdFury VrRoom to 4k120  by editing EDID using their software and upload it to the device? Quite strange for a such expensive hardware. I tested only Linker 4K and it allowed that (but it's not 4k120 capable). Unfortunately I know only one solution for such setup without compromises, but it requires specific hardware: rootable TV LG Webos with piccap and HyperHDR for captured HDR stream correction. BTW it seems that new root method for newer LG firmwares is around the corner.

**Reply by inferno1231** [reply]
Not sure about VrRoom, here is answer from HDFury support: 
![2sasFvW](https://user-images.githubusercontent.com/25138913/178155302-f062cc9a-8064-4f05-9571-1fdcb98f0b0e.png)

Yeah.. I know about LG, interesting to hear about possibility for the new firmware, gotta see how it will work out. 

-----
UPD. Fegarding Vrroom, as far as I understand, I can indeed force it to handle 4k 120hz, but there are no capture cards that support such input. And Vrroom cannot change frame rate. The only thing I think now is that it can downscale resolution, so maybe it is possible to find capture card that supports 1080p 120hz as input for example.

**Reply by awawa-dev** [reply]
Ezcap family for example 320/321/331 supports 1080p120 (but there are quite big differences in the design/quality/video support between these grabbers). Elgato HD60X also promises support for VRR but it's Elgato: didnt test it and it may be not compatible with HyperHDR or some OS (previous model required proprietary codecs for some formats).

**Reply by inferno1231** [reply]
Oh, so there will be another problem with VRR probably, because I am not sure that Hdfury is able to output VRR on one port and remove it on another.

**Reply by awawa-dev** [reply]
I guess hdfury won't be able...so it leaves Elgato HD60X that probably could handle 1080p120 (but not 4k120)  VRR. I've got very nice deal for elgato HD60X in my country. Maybe I give it a try to replace Feintech & Ezcap 321 (but I really love Ezcap 321 HDR colors after tone mapping..).

**Reply by inferno1231** [reply]
Yeah, but one still needs Vrroom to downscale source to 1080p. Damn it is pricey:/

There are some splitters in the wild that claim to split up to 8k 60hz to 2 monitors simultaneously... but here we are again, none of those have integrated downscaler and no capture cards are able to process 4k 120hz input. I think I have to ask Hdfury support some more questions. 

--------
Won't work.

Downscale ANY FRL0/TMDS signals 8K>4K, 4K>2K or 8K>2K up to 120Hz (no downscale for FRL5/VRR tbd)

So no downscale possible for now. Oh well.

**Reply by DariBer** [reply]
@awawa-dev 
> I guess hdfury won't be able...so it leaves Elgato HD60X that probably could handle 1080p120 (but not 4k120) VRR. I've got very nice deal for elgato HD60X in my country. Maybe I give it a try to replace Feintech & Ezcap 321 (but I really love Ezcap 321 HDR colors after tone mapping..).

So if using the Elgato, you don't need a HDMI splitter, like Feintech?

**Reply by awawa-dev** [reply]
If your video signal isn't protected by HDCP then you don't need Feintech.

**Comment by awawa-dev** [comment]
I was expecting Vrroom to handle HDMI2.1/HDMI2.0 compatibility issues, but unfortunately it isn't. And it's limited to 40 Gb/s (HDMI 2.1: 48 Gb/s). At least it's not as broken as my Denon x2700 AVR which the manufacturer refused to repair hardware for their factory broken HDMI2.1 implementation: never again Denon products.

There is good news for the Elgato HD60X: it rocks. The best USB grabber tested so far and is HyperHDR compatible. It even leaves the Ezcap 320 behind. But it's true, it's expensive. Although currently on sale for 140 euro (the Ezcap 320 isn't much cheaper 110 euro, but it does have some downsides). This week I will review it on my blog.

**Comment by lordraiden** [comment]
I'm in a similar situation, I have a LG C2 so 4k 120HZ VRR are mandatory. Under this situation is there any alternative?
Now that HyperHDR support software streaming can we use some APP in the LG TVS to stream to content to hyperHDR?

**Reply by inferno1231** [reply]
> I'm in a similar situation, I have a LG C2 so 4k 120HZ VRR are mandatory. Under this situation is there any alternative? Now that HyperHDR support software streaming can we use some APP in the LG TVS to stream to content to hyperHDR?

Your best bet would be RootMyTV for LG + piccap. If your TV is fairly new / has updated firmware, you will need to wait for RootMyTV v3 to come out.

**Reply by lordraiden** [reply]
ok, on the other side I have a Sony TV so in this case I guess I could use hyperion android grabber
https://github.com/abrenoch/hyperion-android-grabber
Right?
